src/Phansible/RoleManager.php

```diff
@@ -50,7 +50,7 @@ public function getInitialValues()
      */
     public function setupRole(array $requestVars, VagrantBundle $vagrantBundle)
     {
-        array_walk($this->roles, function(Role $role) use($requestVars, $vagrantBundle) {
+        array_walk($this->roles, function (Role $role) use ($requestVars, $vagrantBundle) {
```

Comment:
good! I didn't code serious php for a few months now and I did forget a lot about psr's